diff --git a/src/components/Utilities/Bismillah.svelte b/src/components/Utilities/Bismillah.svelte
index 3183117..f4bb51f 100644
--- a/src/components/Utilities/Bismillah.svelte
+++ b/src/components/Utilities/Bismillah.svelte
@@ -6,7 +6,7 @@
 	<div class="bismilla">
 		<div class="bismilla-text text-3xl text-slate-400 text-center my-2">
 			<span title="In the name of God, the Gracious, the Merciful."
-				>بِسْمِ اللَّهِ الرَّحْمَنِ الرَّحِيمِ</span
+				>بِسْمِ اللَّهِ الرَّحْمَـٰنِ الرَّحِيمِ</span
 			>
 		</div>
 	</div>
